sdk/cognitivelanguage/ai-language-conversations/samples-dev/sample_analyze_conversation_app.ts

```diff
@@ -0,0 +1,76 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+/**
+ * This sample demonstrates how to analyze user query for intents and entities using
+ * a conversation project with a language parameter.
+ *
+ * @summary Conversational query analysis for intents and entities extraction
+ * @azsdk-weight 50
+ */
+
+import { ConversationAnalysisClient, ConversationalTask } from "@azure/ai-language-conversations"
+import { AzureKeyCredential } from "@azure/core-auth";
+import * as dotenv from "dotenv";
+dotenv.config();
+
+//Get secrets
+//You will have to change these environment variables for the sample to work
+var clu_endpoint = process.env.AZURE_CONVERSATIONS_ENDPOINT;
+var clu_key = process.env.AZURE_CONVERSATIONS_KEY;
+var project_name = process.env.AZURE_CONVERSATIONS_PROJECT_NAME;
+var deployment_name = process.env.AZURE_CONVERSATIONS_DEPLOYMENT_NAME;
+
+var service: ConversationAnalysisClient = new ConversationAnalysisClient(clu_endpoint, new AzureKeyCredential(clu_key));
+
+var body: ConversationalTask = {
+    "kind": "Conversation",
+    "analysisInput": {
+        "conversationItem": {
+            "id": "id__7863",
+            "participantId": "id__7863",
+            "text": "Send an email to Carol about the tomorrow's demo"
+        }
+    },
+    "parameters": {
+        "projectName": project_name,
+        "deploymentName": deployment_name
+    }
+}
+
+//Analyze query
+service.analyzeConversation(body).then(
```

Comment:
Could we define an async main function and await this promise inside it for better readability? Examples can be found in the text library samples.